src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -100,9 +101,30 @@ private static OnnxTransform Create(IHostEnvironment env, ModelLoadContext ctx)
             if (!ctx.TryLoadBinaryStream("OnnxModel", r => modelBytes = r.ReadByteArray()))
                 throw env.ExceptDecode();
 
-            var inputColumn = ctx.LoadNonEmptyString();
-            var outputColumn = ctx.LoadNonEmptyString();
-            var args = new Arguments() { InputColumn = inputColumn, OutputColumn = outputColumn };
+            bool isMultiOutput = ctx.Header.ModelVerReadable > 0x00010001;
```

Comment:
>isMultiOutput [](start = 17, length = 13)

supportsMultiInputOutput?

Reply:
done

---
In reply to: [232843660](https://github.com/dotnet/machinelearning/pull/1586#discussion_r232843660) [](ancestors = 232843660)